kuka_agilus_support/launch/test_kr6_r700-sixx.launch.py

```diff
@@ -25,16 +25,17 @@ def generate_launch_description():
             " ",
             PathJoinSubstitution(
                 [FindPackageShare("kuka_agilus_support"),
-                 "urdf", "kr6_r700_sixx.urdf.xacro"]
+                 "urdf", "kr6_r700-sixx.urdf.xacro"]
```

Comment:
while IIRC, legal, using dashes in filenames is not according to conventions.

Underscores tend to be used everywhere.

Reply:
Thanks, I will correct this in all places